src/clj/game/cards/assets.clj

```diff
@@ -1301,21 +1301,27 @@
    (let [ability {:msg "make each player draw 1 card"
                   :label "Make each player draw 1 card (start of turn)"
                   :once :per-turn
-                  :effect (effect (draw 1) (draw :runner))}]
+                  :async true
+                  :effect (effect (draw eid 1 nil)
```

Comment:
Could this be `(req (wait-for (draw state :corp 1 nil) (draw :runner eid 1 nil)))`?